scene/resources/audio_stream_sample.cpp

```diff
@@ -650,7 +650,7 @@ void AudioStreamSample::_bind_methods() {
 
 	BIND_ENUM_CONSTANT(LOOP_DISABLED);
 	BIND_ENUM_CONSTANT(LOOP_FORWARD);
-	BIND_ENUM_CONSTANT(LOOP_PING_PONG);
+	BIND_ENUM_CONSTANT(LOOP_PINGPONG);
```

Comment:
I'm curious is **PINGPONG** the more popular spelling compared to **PING_PONG**?

Reply:
Since @Chaosus 's implementation of the math function is `pingpong()`, it is easy to see when mixed with other enumerated values, I decided to use `pingpong()`, `XXX_PINGPONG`, and `pingponged`.
However, since I am not so particular about it, I can consider using `XXX_PING_PONG` as the enumerated type and `ping_pong()` as the math function.